tsdb/operations/default.json

```diff
@@ -74,7 +78,11 @@
 {
   "name": "date-histo-entire-range",
   "operation-type": "search",
+  {%- if ingest_mode is defined and ingest_mode == "data_stream" %}
```

Comment:
This is required otherwise when running with ingest_mode data_stream this search returns no results.